src/lookupDomains.ts

```diff
@@ -0,0 +1,85 @@
+import { capture } from '@snapshot-labs/snapshot-sentry';
+import { FetchError, isSilencedError } from './addressResolvers/utils';
+import { Address, graphQlCall } from './utils';
+import { isAddress } from '@ethersproject/address';
+
+const ENS_GRAPHQL_URL = 'https://api.thegraph.com/subgraphs/name/ensdomains/ens';
```

Comment:
Could we allow network as input? to get domains owned on sepholia for testnet spaces? not needed for now?

Reply:
Good idea